sdk/monitor/query/MIGRATION.md

```diff
@@ -0,0 +1,211 @@
+# Guide to migrate from `azquery` to `azlogs`, `azmetrics`, and `armmonitor`
+
+With the service team's creation of a new data plane metrics endpoint, the Go SDK team decided to reimagine our Monitor story. After careful deliberation, we decided to restructure Monitor Query to more accurately reflect the [REST API][rest_api].
+
+Logs largely remained the same; the code just moved from `azquery` to [`azlogs`][azlogs].
+
+Metrics has a larger split. Users wanting to use the existing ARM API's should use the [`armmonitor`][armmonitor] module. `armmonitor` contains all the metrics functionality from `azquery` and more.
+
+[`azmetrics`][azmetrics] contains the [new data plane metrics][azmetrics_blog] functionality. This new code was not previously generally available in `azquery`. The code allows users to query data from one or multiple resource IDs using a data plane endpoint. `azmetrics` will expand in the future when the service adds more features.
+
+### Name changes
+
+| old module   | old method name |new module | new method name | 
+| ----------- | ----------- | --- | --- |
+| `azquery` | LogsClient.QueryWorkspace | `azlogs` | Client.QueryWorkspace |
+|  | LogsClient.QueryResource |  | Client.QueryResource |
+| | LogsClient.QueryBatch | | N/A |
+| | MetricsClient.QueryResource | `armmonitor` | MetricsClient.List |
+| | MetricsClient.NewListDefinitionsPager  | | MetricDefinitionsClient.NewListPager |
+| | MetricsClient.NewListNamespacesPager  | | MetricNamespacesClient.NewListPager |
+|  | N/A | `azmetrics` | Client.QueryResources | 
+
+The `azlogs` module does not contain the `QueryBatch` method. To request that functionality, please [file an issue in our github repo][github_issues], so we can prioritize adding it to `azlogs`.
```

Comment:
Is this something we already plan to do?

Reply:
@scottaddie Is the service team still planning to deprecate Batch Query? Was not going to add it to the sdk at all if they are. 

If not, was thinking on waiting to see if there's any demand for batch. It's an ugly API.

Reply:
fixed in [b994525](https://github.com/Azure/azure-sdk-for-go/pull/22753/commits/b994525bed1636c83155a3932ce00aeb00e2fc70)

removing paragraph